python/cusignal/filtering/filtering.py

```diff
@@ -760,10 +760,8 @@ def sosfilt(
     return out
 
 
-_hilbert_kernel = cp.ElementwiseKernel(
-    "",
-    "float64 h",
-    """
+_hilbert_kernel = {
```

Comment:
I like the convention, but it can be pretty verbose. Let's utilize typecasting by changing `float64` to `T`.